test/test_array_ops.cpp

```diff
@@ -157,6 +157,40 @@ TEST(ArrayOps, Unpack) {
   }  // end of for loop
 }  // end of testing Unpack
 
+TEST(ArrayOps, Fill) {
+  std::vector<std::vector<int64>> input_sizes;  // 1-D or higher
+
+  input_sizes.push_back({2, 3, 4});
+  input_sizes.push_back({10, 10, 10});
+  input_sizes.push_back({1, 5});
+  input_sizes.push_back({0});
+  input_sizes.push_back({2, 5, 1, 3, 1});
+
+  vector<int> static_input_indexes = {0};  // has static input
+
+  for (auto const& input_size : input_sizes) {
+    Scope root = Scope::NewRootScope();
+
+    int input_dim = input_size.size();
+    Tensor shape(DT_INT32, TensorShape({input_dim}));
+    AssignInputIntValues(shape, 5);
+
```

Comment:
I think this is not doing what we want.
Lets say: ```input_size = {2,3,4}```
Then, ```input_dim = 3```
Then ```shape``` is a int32 tensor of dims ```1x3``` (or just 3, not exactly sure)
Then ```AssignInputIntValues(shape, 5);``` is saying fill up this ```1x3``` tensor names ```shape``` with values starting from 0 and increasing by int steps. So ```shape``` tensor would now be ```{0,1,2}```.

Which means we will create a tensor of size ```0,1,2``` and not ```{2,3,4}``` as originally intended.

Can you please check and confirm this.